content/runner/lifecycle-hooks.md

```diff
@@ -36,6 +36,7 @@ const { test } = require('@japa/runner')
 test('add two numbers', () => {
   console.log('executed in the test')
 })
+// highlight-start
```

Comment:
There should be an accompanying `highlight-end` comment too.

Reply:
Done